tests/test-admin-locking.sh

```diff
@@ -25,10 +25,10 @@ set -euo pipefail
 setup_os_repository "archive-z2" "syslinux"
 
 # If parallel is not installed, skip the test
-if ! parallel --help >/dev/null 2>&1; then
-    echo "1..0 # SKIP no /usr/bin/parallel"
+if ! which parallel >/dev/null 2>&1; then
```

Comment:
Why is this use of `which` better?

This is a regression, at least for Debian: there are two at-least-semi-common `parallel` utilities (GNU parallel and moreutils parallel), and we specifically want GNU parallel.